source/ccc-lib/http.js

```diff
@@ -5,10 +5,20 @@ import {ONE_MINUTE} from './constants.js'
 export const USER_AGENT = 'ccc-server/0.2.0'
 
 export const http = ky.extend({
-	headers: {
-		'User-Agent': USER_AGENT,
-	},
+	headers: {'User-Agent': USER_AGENT},
 	timeout: 30_000,
+	// hooks: {
```

Comment:
Can we use env variables to be able to turn on or off the interceptor debugging?

Reply:
I didn't feel like doing that here. Can we do that in another PR? 😅

Reply:
Sure it can wait.

Would this work in the follow-up? I know we have to add a module.exports for env.development.plugins transform-inline-environment-variables to include ENABLE_HOOKS

```js
const hooks = process.env.ENABLE_HOOKS === 'true'
  ? {
      beforeRequest: [
        (request) => {
          console.log(`${request.method} ${request.url}`)
        },
      ],
      afterResponse: [
        (_request, _, response) => {
          console.log(`got ${response.url}`)
        },
      ],
    }
  : {}

export const http = ky.extend({
  headers: { 'User-Agent': USER_AGENT},
  timeout: 30_000,
  hooks,
})
```